Source/Core/DiscIO/VolumeGC.h

```diff
@@ -76,6 +78,11 @@ class VolumeGC : public VolumeDisc
                                                     // (only one for BNR1 type)
   };
 
+  struct BootID
+  {
+    u32 id;  // BTID
+  };
+
```

Comment:
This is used in exactly one place, and it doesn't exactly improve clarity there over just reading into a `u32 boot_id`. Do you really need this struct?

Reply:
It will be used later to implement reading the true id's from boot.id, but I can remove it until I implement that

Reply:
I wasn't able to figure out how to implement ReadFile without using a struct, so that's why it's there.

Reply:
Note-- I've fixed this, but I use the struct in pt.2 [here](https://github.com/dolphin-emu/dolphin/pull/10207/files#diff-c817905baf4f25b70eee8d32352e8db45de47d3a5432eaa281bbc805dc25fe6cR80-R85), and that'll be quite a bit harder to work around. Should I keep it like this and try and find a way to rewrite the other code, or should i move this back to a struct, seeing as it'll be used in the next pr.